### ISSUE TITLE
Add only access level on iOS?

### ISSUE THREAD

**Issue by ruskakimov** [CarnegieTechnologies/gallery_saver#126]
Would it be possible to switch to `NSPhotoLibraryAddUsageDescription` permission?

Since it looks like the plugin doesn't require reading from the gallery.

https://github.com/CarnegieTechnologies/gallery_saver/blob/6eb20cde05176fa670d1514710fa047703501536/ios/Classes/SwiftGallerySaverPlugin.swift#L44
Here a [`for` parameter](https://developer.apple.com/documentation/photokit/phphotolibrary/3616052-authorizationstatus) can be passed with [add only access level](https://developer.apple.com/documentation/photokit/phaccesslevel/addonly).

**Comment by ruskakimov** [comment]
@jelenalecic, can you have a look? 🙏

**Comment by Ec7i9se** [comment]
Yes pls, it's impossible to save photo if user granted "only selected photos" or "add only" access. It's critical error!

**Comment by lecicdjuro** [comment]
@ruskakimov please contribute and make an PR fot this
